client_test.go

```diff
@@ -2564,8 +2574,8 @@ func Test_Client_InsertTriggersImmediateWork(t *testing.T) {
 	ctx := context.Background()
 	require := require.New(t)
 
-	ctx, cancel := context.WithTimeout(ctx, 10*time.Second)
-	t.Cleanup(cancel)
+	// ctx, cancel := context.WithTimeout(ctx, 10*time.Second)
+	// t.Cleanup(cancel)
```

Comment:
Mistakenly commented?

Reply:
Oops yep. Actually we should talk about this, but the reason I had it commented is that in case your program is actually stuck, the canceled context supersedes `go test`'s `-timeout` option, so instead of `-timeout` stopping the test of a full stack trace of every goroutine that was still running, you end up with a canceled test with no debugging information.

So a long way of saying that tighter `-timeout` parameters are a better option than context timeouts when it comes to use in tests, but in the interest of not getting into that now, I just uncommented these lines.